keyboards/kprepublic/bm980rgb/rules.mk

```diff
@@ -0,0 +1,25 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
+# Build Options
+#   change yes to no to disable
+#
+#BOOTMAGIC_ENABLE = no       # Enable Bootmagic Lite
+#MOUSEKEY_ENABLE = no        # Mouse keys
+#EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+#NKRO_ENABLE = no            # USB Nkey Rollover
```

Comment:
```suggestion
BOOTMAGIC_ENABLE = no       # Enable Bootmagic Lite
MOUSEKEY_ENABLE = no        # Mouse keys
EXTRAKEY_ENABLE = no        # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = no            # USB Nkey Rollover
```

Reply:
These are specified in info.json

Reply:
![image](https://user-images.githubusercontent.com/43021794/140295096-4e636b9f-e37f-450d-8ce1-c3327a42d77e.png)

Reply:
I shall go disable extrakey then

Reply:
Just pick one place to set it...

Reply:
Already picked, I'll go delete these items in rules.mk then

Reply:
Wait, nevermind, I should have everything in rules.mk if I have anything at all in there

Reply:
Done